### ISSUE TITLE
MONDO:0004769 orbital plasma cell granuloma- Split this Mondo ID?

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#7257]
**Mondo term (ID and Label):**
MONDO:0004769 orbital plasma cell granuloma

**Xref that should be fixed (ID and label):**
Some x-refs/exact matches on this term map to a concept with a different name/CUI in UMLS, Not sure if these are different, or if UMLS has split unnecessarily?
http://identifiers.org/snomedct/80698001 is "orbital myositis"
[Orbital Myositis](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C117296?lang=en)[NCIT]  C117296  "Rare form of myositis that affects only the orbital muscles."
(See also  MESH D055622) All these map to C2350476

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Other comments:**
I am wondering about the pref name on this term, given the other exact matches:
[mesh/D016727](http://identifiers.org/mesh/D016727) is "Orbital pseudotumor" with syn of "Inflammatory Pseudotumor of Orbit"
SNOMEDCT [72789009](http://identifiers.org/snomedct/72789009) is "Inflammatory pseudotumor of orbit"  (both of which are on C0085270 in UMLS)
(Note also that UMLS has split Plasma Cell Granuloma, Orbital into a separate CUI: C2350387, so even though MeSH uses all of these names on D016727, that is split across multiple CUIs)

**Comment by nicolevasilevsky** [comment]
Adding the label for each equiv xref below

equiv xref | label
-- | --
xref: DOID:9369 {source="MONDO:equivalentTo"} | orbital plasma cell granuloma
xref: ICD9:376.12 {source="MONDO:equivalentTo", source="MONDO:i2s"} | Orbital myositis
xref: MESH:D016727 {source="DOID:9369", source="MONDO:equivalentTo"} | Orbital Pseudotumor
xref: NCIT:C117296 {source="MONDO:equivalentTo"} | Orbital Myositis
xref: SCTID:72789009 {source="DOID:9369", source="MONDO:equivalentTo"} | Inflammatory pseudotumor of orbit proper (disorder)
xref: SCTID:80698001 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"} | Orbital myositis (disorder)

**Comment by nicolevasilevsky** [comment]
According to this [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9289764/#:~:text=Orbital%20myositis%20was%20first%20described,subtype%20of%20orbital%20inflammatory%20syndrome.), orbital myositis is an orbital pseudotumor and in 1930 renamed as **benign idiopathic autoimmune inflammatory disease** primarily involving one or more extraocular muscles.

@katiermullen can I reassign this to you and can you please research the proper representation of these concepts? Thanks!

**Comment by katiermullen** [comment]
I am not an ophthalmologist, but I think that the MESH structure makes sense:

Orbital disease

-Orbital myositis D055622 Inflammation of the extraocular muscle of the eye. It is characterized by swelling which can lead to ischemia, fibrosis, or [ORBITAL PSEUDOTUMOR](https://meshb.nlm.nih.gov/record/ui?ui=D016727).

-Orbital pseudotumor D016727 - A nonspecific tumor-like inflammatory lesion in the [ORBIT](https://meshb.nlm.nih.gov/record/ui?ui=D009915) of the eye. It is usually composed of mature [LYMPHOCYTES](https://meshb.nlm.nih.gov/record/ui?ui=D008214); [PLASMA CELLS](https://meshb.nlm.nih.gov/record/ui?ui=D010950); [MACROPHAGES](https://meshb.nlm.nih.gov/record/ui?ui=D008264); [LEUKOCYTES](https://meshb.nlm.nih.gov/record/ui?ui=D007962) with varying degrees of [FIBROSIS](https://meshb.nlm.nih.gov/record/ui?ui=D005355). 

Note: Orbital pseudotumors are often associated with inflammation of the extraocular muscles ([ORBITAL MYOSITIS](https://meshb.nlm.nih.gov/record/ui?ui=D055622)) or inflammation of the lacrimal glands ([DACRYOADENITIS](https://meshb.nlm.nih.gov/record/ui?ui=D003607)).

Also note, I could not find the term "orbital plasma cell granuloma" in PubMed. 

I found "orbital myositis typically occurs in an idiopathic acute form" PMID:31261163 so I don't think it should be considered a subclass of "chronic orbital inflammation."

Bottom line, I agree with splitting the term "orbital plasma cell granuloma" into "orbital pseudotumor" and "orbital myositis."

@nicolevasilevsky @kanems let me know if this makes sense or not. Thanks!

**Comment by nicolevasilevsky** [comment]
thanks @katiermullen! Let's proceed with the proposal to split "orbital plasma cell granuloma" into "orbital pseudotumor" and "orbital myositis."

**Comment by nicolevasilevsky** [comment]
We'd have these two terms:

### orbital plasma cell granuloma

equiv xref | label
-- | --
xref: DOID:9369 {source="MONDO:equivalentTo"} | orbital plasma cell granuloma
xref: MESH:D016727 {source="DOID:9369", source="MONDO:equivalentTo"} | Orbital Pseudotumor
xref: SCTID:72789009 {source="DOID:9369", source="MONDO:equivalentTo"} | Inflammatory pseudotumor of orbit proper (disorder)
xref: UMLS:C0085270 {source="DOID:9369", source="MONDO:notFoundInDiseaseSubset"} | Inflammatory pseudotumor of orbit

### orbital myositis

xref | label
-- | --
xref: ICD9:376.12 {source="MONDO:equivalentTo", source="MONDO:i2s"} | Orbital myositis
xref: NCIT:C117296 {source="MONDO:equivalentTo"} | Orbital Myositis
xref: SCTID:80698001 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"} | Orbital myositis (disorder)
xref: UMLS:C2350476 {source="NCIT:C117296", source="MONDO:notFoundInDiseaseSubset"} | [Orbital Myositis](https://uts.nlm.nih.gov/uts/umls/concept/C2350476)

**Comment by nicolevasilevsky** [comment]
Thank you both! @katiermullen - I will take care of this change
